.gitlab-ci.yml

```diff
@@ -1,36 +0,0 @@
-before_script:
-  - $wololo
```

Comment:
I sure as fuck didn't add that line
https://gitlab.com/d3athrow/vgstation/commit/31d8ae15a2834c7cdf61a5dc63c275191f926d00
Pomf did for arcane reasons